packages/protocol/contracts/layer1/preconf/impl/PreconfWhitelist.sol

```diff
@@ -13,6 +13,9 @@ contract PreconfWhitelist is EssentialContract, IPreconfWhitelist {
     // Tracks the total number of operators in the whitelist
     uint256 public operatorCount;
 
+    // Tracks whether an address is an operator
+    mapping(address operator => bool isOperator) public isOperator;
+
```

Comment:
`uint256[50] private __gap;` below should be changed to `uint256[47] private __gap;`